dlk/python/dlk/plugins/tf.py

```diff
@@ -61,159 +53,149 @@
     'DT_COMPLEX64': None,
     'DT_COMPLEX128': None,
     's': String(),
-
-    # custom vector
     'STRINGS': None,
-
-    # struct
     'TENSOR': None,
     'GRAPH': None,
     't': None,
     'g': None,
-
-    # struct vector
     'TENSORS': None,
     'GRAPHS': None,
 }
 
+
 DLK_OPERATOR_MAP: Dict[str, str] = {
     'Conv2D': 'Conv',
     'FusedBatchNorm': 'BatchNormalization',
     'AvgPool': 'AveragePool',
     'BiasAdd': 'Add',
-    'ConcatV2': 'ConcatOnDepth'
+    'ConcatV2': 'ConcatOnDepth',
+    'Split': 'Split',
 }
 
 
-class Node(object):
-    ATTRIBUTE_TYPE_MAP = {
-        0: 'UNDEFINED',
-        1: 'FLOAT',
-        2: 'INT',
-        3: 'STRING',
-        4: 'TENSOR',
-        5: 'GRAPH',
-        6: 'FLOATS',
-        7: 'INTS',
-        8: 'STRINGS',
-        9: 'TENSORS',
-        10: 'GRAPHS',
-    }
+TF_ATTR_TYPE_MAP: Dict[str, str] = {
+    'padding': 's',
+    'data_format': 's',
+    '_gradient_op_type': 's',
+    'strides': 'list.i',
+    'ksize': 'list.i',
+    'epsilon': 'f',
+    'is_training': 'b',
+    'use_cudnn_on_gpu': 'b',
+    'transpose_b': 'b',
+    'transpose_a': 'b',
+    'block_size': 'i',
+    'num_split': 'i',
+    'N': 'i',
+    '_output_shapes': 'list.shape',
+    'T': 'type',
+    'Tshape': 'type',
+    'Tpaddings': 'type',
+    'Tidx': 'type',
+    'dtype': 'type',
+    'shape': 'shape.dim',
+    'value': 'tensor',
+    '_class': 'list.s',
+}
 
-    ATTRIBUTE_VALUE_MAP = {
-        1: 's',
-        2: 'i',
-        3: 'f',
-        4: 'b',
-        5: 'type',
-        6: 'shape',
-        7: 'tensor',
-        8: 'list',
-        9: 'func',
-        10: 'placeholder',
-    }
 
-    def __init__(self, op_nd) -> None:  # type: ignore
-        self.nd_ = op_nd
-        self.attributes = []  # type: ignore
-        for key in self.nd_.attr.keys():
+class TFProtoWrapper(object):
+    DATA_TYPE_MAP = {v: k for k, v in types_pb2.DataType.items()}
+
+    def __init__(self, tf_node) -> None:
+        self.tf_nd = tf_node
```

Comment:
When I first see tf_nd in this code,  I had thought `nd` is n-dimensional array.
Did you read `The Art of Readable Code` ? 
This abbreviations looks bad for new comer. `tf_mp` and `tf_gp` and `lst`and `dic` also.

Reply:
I agree they are bad name by any means, and before that, I forgot to revert these changes in `tf.py`, I think they shouldn't be included in this PR.